.flake8/azul_flake8.py

```diff
@@ -29,6 +29,11 @@
     config,
 )
 
+# `scripts/` is not in PYTHONPATH because scripts should not be imported from
+# other locations in the project. However, files in `scripts/` may import from
+# each other. In order to determine the proper ordering of these intra-script
+# imports, we need to make them resolvable from outside of `scripts/` where the
+# flake8 command will presumably be run.
```

Comment:
I trust your opinion here, but based on my understanding, this seems more correct:

```suggestion
# imports, we need to make them resolvable from within this plugin.
```